README.md

```diff
@@ -1,39 +1,98 @@
-<p align="center">
-  <b>ConnectALL</b><br>
-  <i>Platform to help close the Digital Divide gap in collaboration with the White House</i><br>
-  <a href="http://make.sc/whitehouse">The Initiative</a> |
-  <a href="https://www.makeschool.com/founding-class">Make School</a>
-  <!--<a href="#">Press Kit</a>-->
-  <br><br>
-  <img src="https://www.whitehouse.gov/profiles/forall/modules/custom/gov_whitehouse_www/images/icons/wh_logo_seal.png" height="100" />
-</p>
+#[Go Digital with Us](http://www.godigitalwith.us/)
+![ruby version](https://img.shields.io/badge/ruby-2.3.1-brightgreen.svg)
+
+<center>
```

Comment:
Yeah I found some github project that did some crazy stuff with their readme

On Tue, Jul 26, 2016 at 7:56 PM, Dan Kozlowski <notifications@github.com> wrote:
In README.md [https://github.com/cmaher92/ConnectAll/pull/49#discussion_r72373650] :

> @@ -1,39 +1,98 @@
> -<p align="center">
> -  <b>ConnectALL</b><br>
> -  <i>Platform to help close the Digital Divide gap in collaboration with the White House</i><br>
> -  <a href="http://make.sc/whitehouse">The Initiative</a> |
> -  <a href="https://www.makeschool.com/founding-class">Make School</a>
> -  <!--<a href="#">Press Kit</a>-->
> -  <br><br>
> -  <img src="https://www.whitehouse.gov/profiles/forall/modules/custom/gov_whitehouse_www/images/icons/wh_logo_seal.png" height="100" />
> -</p>
> +#[Go Digital with Us](http://www.godigitalwith.us/)
> +![ruby version](https://img.shields.io/badge/ruby-2.3.1-brightgreen.svg)
> +
> +<center>


Whoa, this works in Markdown?

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub [https://github.com/cmaher92/ConnectAll/pull/49/files/e9586f665656d4e5be60521c0eae338f0edb8733#r72373650] , or mute the thread [https://github.com/notifications/unsubscribe-auth/AJ2MQppI9ww0DSAK_msVYcxXh4HZPOjIks5qZsjQgaJpZM4JTrDI] .